sunpy/net/vso/vso.py

```diff
@@ -966,4 +964,4 @@ def get(query_response, path=None, methods=('URL-FILE',), downloader=None):
     return g_client.get(query_response, path, methods, downloader)
 
 
-get.__doc__ = VSOClient.get.__doc__
+get.__doc__ = VSOClient.search.__doc__
```

Comment:
Why changing the `get` doc to the `search` one?

Reply:
because `VSOClient.get` now has no actual docs, because it's now deprecated.